src/beamlime/applications/handlers.py

```diff
@@ -427,9 +457,11 @@ def __init__(
         logger: BeamlimeLogger,
         image_path_prefix: ImagePath,
         max_column: MaxPlotColumn = DefaultMaxPlotColumn,
+        normalize: NormalizeHistogramFlag = DefaultNormalizeHistogramFlag,
```

Comment:
Same here, as mentioned yesterday on Slack, please undo all changes to plotting, as I already have a ton of changes here.